pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.simple</groupId>
     <artifactId>jdub_2.11</artifactId>
-    <version>1.5.0-SNAPSHOT</version>
+    <version>1.6.0-SNAPSHOT</version>
```

Comment:
we dont need to bump this, as we'll release these changes with 1.5.x

Reply:
Okay, I'll drop that commit